derrida/books/views.py

```diff
@@ -239,6 +239,53 @@ def get_queryset(self):
                 solr_facet = self.form.solr_field(facet)
                 # filter the query: facet matches any of the terms
                 sqs = sqs.filter(**{'%s__in' % solr_facet: search_opts[facet]})
+        # request range facets for References, adapated from logic
+        # above for Instances
+        # get max/min from database to specify range start & end values
+        # TODO: should probably cache max/min values so we don't have
+        # to calculate them every time
+        # NOTE: restricting to cited books currently returns null for copyright
```

Comment:
Is this note still the case here?  If we're querying by reference objects I think we already are restricting to cited books...